examples/ingest-feedback-events/1/widget-angular.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
How come these examples don't have project_id? Isn't that a required field in user feedback? It's not on the schema either.